### PR TITLE
RHMAP-10094 update aerogear-ios-sync to Swift 2.3

### PR DESCRIPTION
**Pull request by jcesarmobile**
Ran the migration tool and no code changes were needed
Updated README.md
Set Deployment Target to iOS 8.0

### CODE REVIEW

**Comment by jcesarmobile** [comment]
@corinnekrych can you review?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # aerogear ios sync [![Build Status](https://travis-ci.org/aerogear/aerogear-ios-sync.png)](https://travis-ci.org/aerogear/aerogear-ios-sync) [![Pod Version](http://img.shields.io/cocoapods/v/AeroGearSync.svg?style=flat)](http://cocoadocs.org/docsets/AeroGearSync/)
 
-> This module currently build with Xcode 7 (Swift 2.0) and supports iOS8, iOS9.
+> This module currently build with Xcode 7.3.1 or Xcode 8 (Swift 2.3) and supports iOS8, iOS9.
```
Comment (corinnekrych) [inline README.md]: and iOS10

**Comment by corinnekrych** [comment]
@jcesarmobile Review done in: https://github.com/jcesarmobile/aerogear-ios-sync/pull/1
i've removed warning in Xcode 7.3.1 / Xcode8 to remove deprecate warning

**Comment by jcesarmobile** [comment]
Merged your changes and updated README.md with iOS 10

**Pull request merged**